### PR TITLE
Minor revisions to text and links

### PR DESCRIPTION
**Pull request by DelfinGala**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by andrew-glenn — APPROVED** [review]

File: .specific/_settings.adoc [hunk 1]
```diff
@@ -25,11 +25,11 @@
 // :marketplace_listing_url: https://example.com/
 // Uncomment the following attribute to add a statement about AWS and our stance on compliance-related Quick Starts. 
 // :compliance-statement: Deploying this Quick Start does not guarantee an organization’s compliance with any laws, certifications, policies, or other regulations.  
-// Uncomment the following attribute if you are deploying a CDK Quick Start. Make sure to comment out :parameters_as_appendix: also.
+// Uncomment the following attribute if you are deploying a CDK Quick Start. Ensure to comment out :parameters_as_appendix: also.
```
Comment (MarciaRieferJohnston) [inline .specific/_settings.adoc:28]:
```suggestion
// Uncomment the following attribute if you are deploying a CDK Quick Start. Comment out :parameters_as_appendix: also.
```

File: .specific/_settings.adoc [hunk 2]
```diff
@@ -25,11 +25,11 @@
...
 // :cdk_qs:
-// Uncomment the following attribute if you are deploying a Terraform Quick Start. Make sure to comment out :parameters_as_appendix: also.
+// Uncomment the following attribute if you are deploying a Terraform Quick Start. Ensure to comment out :parameters_as_appendix: also.
```
Comment (MarciaRieferJohnston) [inline .specific/_settings.adoc:30]:
```suggestion
// Uncomment the following attribute if you are deploying a Terraform Quick Start. Comment out :parameters_as_appendix: also.
```

File: .specific/_settings.adoc [hunk 3]
```diff
@@ -25,11 +25,11 @@
...
 // :terraform_qs:
-// Uncomment the following two attributes if you are deploying a Terraform Quick Start. Make sure to comment out :parameters_as_appendix: also.
+// Uncomment the following two attributes if you are deploying a Terraform Quick Start. Ensure to comment out :parameters_as_appendix: also.
```
Comment (MarciaRieferJohnston) [inline .specific/_settings.adoc:32]:
```suggestion
// Uncomment the following two attributes if you are deploying a Terraform Quick Start. Comment out :parameters_as_appendix: also.
```

File: .specific/deploy_steps.adoc [hunk 4]
```diff
@@ -20,7 +20,7 @@ endif::marketplace_subscription[]
...
+WARNING: If you deploy {partner-product-short-name} into an existing VPC, ensure that your VPC has two private subnets in different Availability Zones for the workload instances and that the subnets are not shared. This Quick Start does not support https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[shared subnets^]. The subnets require https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html[NAT gateways^] in their route tables to allow the instances to download packages and software without exposing the instances to the internet. Also ensure that the domain name in the DHCP options is configured, as explained in http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_DHCP_Options.html[DHCP options sets^]. Provide your VPC settings when you launch the Quick Start.
```
Comment (MarciaRieferJohnston) [inline .specific/deploy_steps.adoc:23]: @DelfinGala I'm curious about undoing the contractions. The AWS Style Guide lists "you're" and "aren't" and "doesn't" as acceptable.

**Review by MarciaRieferJohnston — CHANGES_REQUESTED** [review]

**Review by MarciaRieferJohnston — APPROVED** [review]

**Review by andrew-glenn — APPROVED** [review]

**Pull request merged**